libs/payments/metrics/src/lib/glean/glean.types.ts

```diff
@@ -13,7 +14,7 @@ export const PaymentProvidersTypePartial = [
   'external_paypal',
 ] as const;
 export type PaymentProvidersType =
-  | 'card'
+  | Stripe.PaymentMethod.Type
```

Comment:
`card` is a subset of the payment method types. Do we want to allow non-card payment methods to pass?

Reply:
This is only a typing, so doesn't have impact on runtime behavior.